### DISCUSSION TITLE
router.options.generator_class is no longer available in Symfony 4. So what is the replacement ?

### DISCUSSION THREAD

**Discussion by chieuvan9283** [symfony/symfony#51260] [Q&A]
Hi Community, I am upgrading from Symfony 2 to Symfony 4 and running into this issue You have requested a non-existent parameter router.options.generator_class . I understand that has been removed in Symfony 4. So what is the replacement for router.options.generator_class ? Thank you

**Comment by MatTheCat** [comment]
Hello, I think https://github.com/symfony/symfony/issues/30833#issuecomment-531568131 is still relevant.
